src/gc/gcinterface.d

```diff
@@ -148,6 +152,13 @@ interface GC
      */
     core.memory.GC.Stats stats() nothrow;
 
+    /**
+     * Track beginning and end of allocation
+     * Useful for debugging and tuning.
+     */
+    void monitor (CollectionStartHook on_start, CollectionEndHook on_end)
+        nothrow @nogc;
```

Comment:
`@__future`?

Reply:
That makes sense.
I would be extremely surprised if someone was implementing this interface, but I suppose that's a good test anyway.